definitions/npm/testcafe_v0.x.x/flow_v0.47.x-/testcafe_v0.x.x.js

```diff
@@ -643,12 +643,11 @@ declare module 'testcafe' {
     declare module.exports: {
         (hostname: string, port1: number, port2: number): Promise<TestCafe>,
 
-        Selector(init: string | ((...args: any[]) => Node | Node[] | NodeList | HTMLCollection) | TestCafe$SelectorFn | TestCafe$NodeSnapshot | TestCafe$SelectorPromise,
+        Selector(init: string | ((...args: any[]) => null | Node | Node[] | NodeList<*> | HTMLCollection<*>) | TestCafe$SelectorFn | TestCafe$NodeSnapshot | TestCafe$SelectorPromise,
```

Comment:
Added `null` here since the builtins expect either a DOM element or `null` when using `getElementById`